README.rst

```diff
@@ -0,0 +1,730 @@
+Porter
+======
+
+Overview
+--------
+NuCypher Porter can be described as the *“Infura for NuCypher”*. Porter is a web-based service that performs
+nucypher-based protocol operations on behalf of applications.
+
+Its goal is to simplify and abstract the complexities surrounding the nucypher protocol to negate the need for
+applications to interact with it via a python client. Porter introduces the nucypher protocol to cross-platform
```

Comment:
```suggestion
applications to interact with the network directly. Porter introduces the nucypher protocol to cross-platform
```

Reply:
> Porter introduces the nucypher protocol to cross-platform functionality including web and mobile applications.

With the evolution of nucypher-ts this is not the motivation anymore IMO.

Reply:
👍